### PR TITLE
highlight on bulk edge where filtering expression

### LINKED ISSUE #824: Highlight somewhere in docs on property filter in bulk pattern's condition
It's easy for users to write:
> incorrect
```
match path=(n:player{name: "Tim Duncan"})-[e:follow*1..2]-(m) \
    where e.degree > 0 \
    return path
```
rather than:
> correct
```
match path=(n:player{name: "Tim Duncan"})-[e:follow*1..2]-(m) \
    where ALL(e_ in e WHERE e_.degree > 0) \
    return path
```

Could we put the correct expression somewhere(and in a highlighted way) in docs?

### PR DESCRIPTION
**Pull request by wey-gu**
close #824

### CODE REVIEW

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 1]
```diff
@@ -370,6 +370,22 @@ nebula> MATCH (v:player{name:"Tim Duncan"}) -[*0]-> (v2) \
 +----------------------------------------------------+
 ```
 
+!!! Note
+
+    如果想要通过描述多跳的边的过滤条件，比如`-[e:follow*2]->`，这时候 `e` 不再试单条边时候的数据类型了，而是一列边的符合类型，例如：
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:373]:
```suggestion
    如果想要通过描述多跳的边的过滤条件，比如`-[e:follow*2]->`，这时候 `e` 不再是单条边时候的数据类型了，而是一列边组成的列表，例如：
```

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 2]
```diff
@@ -370,6 +370,22 @@ nebula> MATCH (v:player{name:"Tim Duncan"}) -[*0]-> (v2) \
...
+    这是错误的表达，因为`e`是一个边的列表，没有`.degree`的属性
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:374]:
```suggestion
    以下语句可以运行但是没有返回数据，因为`e`是一个列表，没有`.degree`的属性。
```

**Comment by wey-gu** [comment]
sorry, the rebase make things a mess, I will fix it via the command line.

File: docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md [hunk 3]
```diff
@@ -368,6 +368,30 @@ nebula> MATCH (v:player{name:"Tim Duncan"}) -[*0]-> (v2) \
...
+```ngql
+nebula> MATCH p=(v:player{name:"Tim Duncan"})-[e:follow*2]->(v2) \
+        WHERE e.degree > 1 \
+        RETURN DISTINCT v2 AS Friends;
+```
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md:379]: If you wish to include the code blocks in the note, indent them as well.

**Review by randomJoe211 — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Comment by abby-cyber** [comment]
already added this note to the English doc. @izhuxiaoqing

**Comment by wey-gu** [comment]
@abby-cyber we should cherry pick this to master and other newer branches :-P

**Comment by abby-cyber** [comment]
> 

Already did :)

**Pull request merged**